editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/navigator/CamelCtxNavContentProvider.java

```diff
@@ -47,17 +47,13 @@
 	@Override
 	public Object[] getChildren(Object parent) { 
 		if(parent instanceof IFile){	
-			return getRoutes((IFile)parent,true);
+			return getRoutes((IFile)parent, false); // FUSETOOLS-2363 -> when doing lazy loading routes will duplicate
```

Comment:
Yeah, lazy loading has been turned off for this workaround. But that might have negative impact on the performance. Since I haven't heard back from GSS about priorities I am tempted to ignore this PR for now and work on a better solution for 10.x instead.